src/Planner/PlannerJoinTree.h

```diff
@@ -16,6 +16,7 @@ struct JoinTreeQueryPlan
     QueryPlan query_plan;
     QueryProcessingStage::Enum from_stage;
     std::set<std::string> used_row_policies;
+    std::vector<ActionsDAGPtr> actions_dags = {};
```

Comment:
Vector has default constructor, so we can omit it.

Reply:
It required to specify member names then.

I replaced

```cpp
return {std::move(query_plan), from_stage, std::move(used_row_policies)};
```

to 
```cpp
return JoinTreeQueryPlan{
        .query_plan = std::move(query_plan),
        .from_stage = from_stage,
        .used_row_policies = std::move(used_row_policies),
    };
```